README.md

```diff
@@ -4,7 +4,7 @@
 | Code style checks | [![Circle CI](https://circleci.com/gh/shadow-robot/sr_core.svg?style=shield)](https://circleci.com/gh/shadow-robot/sr_core) |
 | Unit tests        | [![Build Status](https://img.shields.io/shippable/55df21b21895ca4474103997.svg)](https://app.shippable.com/projects/55df21b21895ca4474103997) |
 | Install tests     | [![Build Status](https://semaphoreci.com/api/v1/projects/23fc083a-c386-4321-90e6-cdb2235c288f/525053/shields_badge.svg)](https://semaphoreci.com/shadow-robot/sr_core) |
-| Code Coverage     | [![codecov.io](https://img.shields.io/codecov/c/github/shadow-robot/sr_core/indigo-devel.svg)](http://codecov.io/github/shadow-robot/sr_core?branch=indigo-devel) |
+| Code Coverage     | [![codecov.io](https://img.shields.io/codecov/c/github/shadow-robot/sr_core/kinetic-devel.svg)](http://codecov.io/github/shadow-robot/sr_core?branch=kinetic-devel) |
```

Comment:
we should remove this badge. Since we are not supporting it correctly.

Reply:
Removed in the latest commit